app/assets/stylesheets/custom.css.scss

```diff
@@ -36,6 +39,11 @@ body {
   padding: 5px;
 }
 
+// navbar dropdown
+.navbar-default .navbar-nav > .open > a:focus {
```

Comment:
Merge rule `.navbar-default .navbar-nav > .open > a:focus` with rule on line 25<br>Selector should have depth of applicability no greater than 2, but was 4